### PR TITLE
Fix: read-only styles missing when in read only mode

### LINKED ISSUE #131: Read-only styles missing for toggle-button
Compare the icons in date picker and time picker in this image.
image

When time picker has `readonly` the icon should use a lighter color (like in date picker above) and it should not have a hover style in that case. Now it also gets darker when hovered even when `readonly` is set.

### PR DESCRIPTION
**Pull request by hongduc-phan**
Fixes #131

### CODE REVIEW

File: test/input-test.html [hunk 1]
```diff
@@ -205,6 +205,11 @@
           });
         });
 
+        it(`should reflect to attribute when readonly property is set`, () => {
+          timePicker['readonly'] = true;
```
Comment (web-padawan) [inline test/input-test.html]: Please follow we style we use elsewhere:

```suggestion
          timePicker.readonly = true;
```

Comment (hongduc-phan) [reply test/input-test.html]: done!

File: test/input-test.html [hunk 2]
```diff
@@ -205,6 +205,11 @@
           });
         });
 
+        it(`should reflect to attribute when readonly property is set`, () => {
```
Comment (Haprog) [inline test/input-test.html]: No need for template literal here, please use regular single quotes instead.
```suggestion
        it('should reflect to attribute when readonly property is set', () => {
```

Comment (hongduc-phan) [reply test/input-test.html]: done

**Pull request merged**